src/framebuffers.h

```diff
@@ -3,13 +3,25 @@
 
 #include <PR/ultratypes.h>
 #include "config.h"
+#include "common_structs.h"
 
-extern u8 D_802BA290[22768];
-extern u8 D_802BFB80[77824];
-extern u8 D_802D2B80[32768];
-extern u8 D_802DAB80[20480];
-extern u8 D_802DFB80[2][2][18688];
-extern u16 D_802F1F80[2][4][2048];
+typedef struct {
+    char unk_00[0x1000];
+} struct_D_802BFB80; // size = 0x1000
+
+typedef struct {
+    char unk_00[0x920];
+} struct_D_802DFB80; // size = 0x920
+
+typedef struct {
+    char unk_00[0x200];
+} struct_D_802F1F80; // size = 0x200
```

Comment:
I'm fine with this change. Is the purpose of it to make it look cleaner?

Reply:
As best as I can tell its closer to being accurate to the actual underlying data. I doubt the underlying data is actually just a struct around a 0x200 sized char array, but its better than a plain char array.

If you look at `func_80027A201` in `code_80027040.c` you can see that `osPiStartDma` tries to copy 0x200 bytes from `_kart_texturesSegmentRomStart` to `D_802F1F80`. So, as best as I can tell, entries in `D_802F1F80` need to be 0x200 bytes large.